vue/src/TagmanagerTrackingCode/TrackingCodeCommon.vue

```diff
@@ -80,12 +80,16 @@
       <div v-if="showStep3">
         <li v-html="$sanitize(getReactStep3)"></li>
       </div>
-      <pre
-        class="codeblock"
-        v-text="installInstruction.embedCode"
-        v-select-on-focus="{}"
-        ref="codeblock"
-      />
+      <div>
+      <div>
```

Comment:
Any reason for duplicating the `div` here?

Reply:
That's a merge artifact, I've removed it.